src/phalanges/core.cljs

```diff
@@ -32,19 +32,23 @@
 ;; Keyword/Keycode conversion
 
 (def
-  ^{:doc "ClojureScript friendly version of goog.events.KeyCodes."}
-  keyword->keycode
+  ^{:doc "ClojureScript friendly version of goog.events.KeyCodes.
+  
+  Produced with following code:
```

Comment:
I think this could probably be documented with a comment above this def instead of in the docstring.